src/scripts/main.js

```diff
@@ -1,7 +1,115 @@
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class.js';
+
+const game = new Game();
 
 // Write your code here
+const gameScore = document.querySelector('.game-score');
+const startButton = document.querySelector('.buttom start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('message-start');
+
+const fields = document.querySelector('.field-cell');
+
+document.addEventListener('keydown', keypressHandler);
+startButton.addEventListener('click', startGame);
+
+function keypressHandler(e) {
+  const keyActions = {
+    Escape: restartGame,
+    Enter: startGame,
+    ArrowUp: () => updateBoard(game.moveUp()),
+    ArrowDown: () => updateBoard(game.moveDown()),
+    ArrowLeft: () => updateBoard(game.moveLeft()),
+    ArrowRight: () => updateBoard(game.moveRight()),
+  };
+
+  // Ações globais (Escape e Enter)
+  if (keyActions[e.key] && (e.key === 'Escape' || e.key === 'Enter')) {
+    keyActions[e.key]();
+
+    return;
+  }
+
+  // Ações de movimento apenas se o jogo estiver em progresso
+  if (game.status() === 'playing' && keyActions[e.key]) {
```

Comment:
The method `game.status()` does not exist. It should be `game.getStatus()` to correctly retrieve the game status.